oemoflex/preprocessing.py

```diff
@@ -0,0 +1,261 @@
+import logging
+import os
+
+import pandas as pd
+
+
+bus_list = [
+    'AT-el-bus',
+    'BE-el-bus',
+    'CH-el-bus',
+    'CZ-el-bus',
+    'DE-el-bus',
+    'DK-el-bus',
+    'FR-el-bus',
+    'IT-el-bus',
+    'LU-el-bus',
+    'NL-el-bus',
+    'PL-el-bus',
+]
+
+country_list = [
+    'AT',
+    'BE',
+    'CH',
+    'CZ',
+    'DE',
+    'DK',
+    'FR',
+    'IT',
+    'LU',
+    'NL',
+    'PL',
+]
+
+link_list = [
+    'AT-CH',
+    'AT-CZ',
+    'AT-IT',
+    'BE-FR',
+    'BE-LU',
+    'BE-NL',
+    'CH-FR',
+    'CH-IT',
+    'CZ-PL',
+    'DE-AT',
+    'DE-BE',
+    'DE-CH',
+    'DE-CZ',
+    'DE-DK',
+    'DE-FR',
+    'DE-LU',
+    'DE-NL',
+    'DE-PL',
+    'DK-NL',
+    'FR-IT',
+    'FR-LU',
+
+]
+
+datetimeindex = pd.date_range(start='2019-01-01', freq='H', periods=8760)
+
+module_path = os.path.dirname(os.path.abspath(__file__))
+
+def create_default_elements_files(
```

Comment:
```suggestion
def create_default_elements(
```

Reply:
To unify function names, let's drop the '_files'. I remember that you suggested this somewhere? Has to be adapted in the script as well.

Reply:
https://github.com/modex-flexmex/oemo-flex/pull/20/commits/8b744b2019b0bb12b125ea9163e06ebe1d5579db resolved this